packages/services/src/services/Translation/Translation.js

```diff
@@ -50,7 +50,18 @@ class TranslationAPI {
           'Content-Type': 'text/plain',
         },
       })
-      .then(response => response.data);
+      .then(response => this.transformData(response.data));
+  }
+
+  /**
+   * Transforms translation data
+   *
+   * @param   {object} data translation data to be transformed
+   * @returns {object} Translation data
+   */
+  static transformData(data) {
+    data.footerMenu.push(data.socialFollow);
```

Comment:
@kennylam can you sync up with @photodow on how you guys want to align on transforming the data when it comes in?

cc: @annawen1

Reply:
@jeffchew we met on Friday to compare notes. There is more we can do to transform I think, but we have issue [#1652](https://github.ibm.com/webstandards/digital-design/issues/1652) we can use to address more detailed stuff. I think we're aligned on the approach for the initial release.